share/models/change.py

```diff
@@ -89,7 +93,7 @@ def accept(self, save=True):
         ret = []
         with transaction.atomic():
             self._changes_cache = list(self.changes.all())
-            for c in self._changes_cache:
+            for c in sorted(self._changes_cache, key=lambda c: c.type):
```

Comment:
Oh, I already changed ChangeGraph to sort merge nodes first, which I think is much better.

(Looks like the `sorted` docs have been unclear [since 2009](https://stackoverflow.com/questions/1915376/is-pythons-sorted-function-guaranteed-to-be-stable))